ibm/resource_ibm_cis_domain_settings.go

```diff
@@ -2,268 +2,1075 @@ package ibm
 
 import (
 	"log"
-	"strings"
 
+	"github.com/IBM/go-sdk-core/v3/core"
+	"github.com/IBM/networking-go-sdk/zonessettingsv1"
 	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+)
 
-	v1 "github.com/IBM-Cloud/bluemix-go/api/cis/cisv1"
+const (
+	cisDomainSettingsDNSSEC                          = "dnssec"
+	cisDomainSettingsWAF                             = "waf"
+	cisDomainSettingsSSL                             = "ssl"
+	cisDomainSettingsUniversalSSL                    = "ssl_universal"
```

Comment:
`cisDomainSettingsUniversalSSL` is this attribute been supported in the resource.. as per our discussion we planned for new resource for tls settings...

Reply:
I forgot to remove this const. Removed it now.